src/firebase.js

```diff
@@ -39,6 +39,10 @@ const stateGroupsCollection = db.collection('stateGroups');
 const usersCollection = db.collection('users');
 const userStatesCollection = db.collection('userListItemStates');
 
+// firebase oAuth login
+const googleOAuthLogin = new firebase.auth.GoogleAuthProvider();
```

Comment:
this is part of another branch. you should migrate this over to the new branch and clean up this one.